lib/diagnostic.js

```diff
@@ -7,22 +7,35 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
+const min = function min(small, medium, large) {
+  small = 1;
+  medium = 2;
+  large = 3;
+  return Math.min(min);
 
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
+const sum = function sum(a, b) {
+  a = 2;
+  b = 2;
+  Math.sum(sum);
```

Comment:
You need to look up the definition and the syntax of these methods before trying to use them. Please study this solution:
```
const sum = function sum() {
  let result = 0;
  for (let i = 0; i < arguments.length; i++) {
    result += arguments[i];
  }

  return result;
};
```